gitops/azure-devops/build.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 function copy_files() {
-    cp -r * $HOME/
+    cd .. && cp -r * $HOME/
```

Comment:
I think this part depends on how they structure the files in their source repo, right?

Reply:
Yes, how do you think we should go about that?

Reply:
Maybe this is something we should add to README in the gitops root folder. I'm assuming right now we're not specifying on how we expect the files to be structured, but if we add this `cd ..` we have to either specify it, or not `cd ..` ?

Reply:
For now, let's just leave it as `cp -r * $HOME/`, since that appears to be working for me too. Will need to make note of it.

Reply:
Sounds great